dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -18,7 +18,8 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-RESTRICT=test
+# Upstream issue https://github.com/haskell/hackage-security/issues/247
+PATCHES="${FILESDIR}/${PN}-0.6.0.1-disable-failing-aeson-canonical-test.patch"
```

Comment:
Not at all a blocker for me - more so advice for next time - but I believe that a PATCHES array is more common and [preferred in Gentoo](https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/), e.g. `PATCHES=( "${FILESDIR}"/${P}-blah.patch )`

Reply:
I knew that the array was *an option*, but I didn't realize it was strongly preferred. I left this is a single string due to I only had a single patch, but I will ask in #gentoo-dev-help

Reply:
I asked, they said use an array.